build fix for ubuntu 20.04

dennisvanderpool opened the pull request:
stropts.h seems not supported on Ubuntu/Linux, something with streams. Replacing with #include <sys/ioctl.h> worked for me.

Source: https://forums.fedoraforum.org/showthread.php?194667-error-stropts-h-No-such-file-or-directory

dennisvanderpool committed:
build fix for ubuntu 20.04

stropts.h seems not supported on Ubuntu/Linux, something with streams. Replacing with #include <sys/ioctl.h> worked for me.

Source: https://forums.fedoraforum.org/showthread.php?194667-error-stropts-h-No-such-file-or-directory

Malvineous commented:
Best not to commit commented-out code, so if you want to remove that line entirely then I'll merge it.  Thanks for the fix!

dennisvanderpool committed:
Removed commented out "#include <stropts.h>"

Code should be clean, not commented out code, so i've removed the "#include <stropts.h>" commented out code as requested by Adam

dennisvanderpool commented:
@Malvineous Done :)

Malvineous commented:
Merged in 36bc864, apologies for the delay!
